devtools.gemspec

```diff
@@ -17,6 +17,11 @@ Gem::Specification.new do |gem|
   gem.extra_rdoc_files      = %w[README.md TODO]
   gem.required_ruby_version = '>= 2.1'
 
+  gem.add_runtime_dependency 'procto', '~> 0.0.x'
+  gem.add_runtime_dependency 'anima', '~> 0.3.x'
+  gem.add_runtime_dependency 'concord', '~> 0.1.x'
+  gem.add_runtime_dependency 'adamantium', '~> 0.2.x'
```

Comment:
Out of a habit, I'd prefer to align the version strings., just like the dependencies below.